### PR TITLE
Testing seer command

### PR DESCRIPTION
**Pull request by neihousaigaai**

### CODE REVIEW

File: json/player_score_list.json [hunk 1]
```diff
@@ -1 +1 @@
-{}
+{"1": 29340, "2": 34860, "3": 41340, "4": 40860, "5": 41760, "6": 1580, "884454526063636590": 160, "420248840583643156": 280, "884452824497422426": 280, "884455004554022912": 280, "884454427765932083": 160}
```
Comment (nitro2) [inline json/player_score_list.json:1]: Please remove this?

File: test.py [hunk 2]
```diff
@@ -66,10 +66,19 @@ async def test_case(game, filepath):
...
+                    action_str, expected_result = action_str.split(" = ")
+
                 author_name, command = action_str.split()[:2]
                 target_name = action_str.split()[2:]
                 text = await game.do_player_action(command, id_map[author_name], *[id_map[i] for i in target_name])
                 print(text)
+
+                if command == "seer":
+                    target_seen_as_werewolf = game.players[id_map[target_name[0]]].seer_seen_as_werewolf()
+                    assert expected_result.capitalize() == str(target_seen_as_werewolf)
```
Comment (nitro2) [inline test.py:80]: Add comment and example please

**Review by nitro2 — APPROVED** [review]
Minor change request, overall ok

File: json/player_score_list.json (hunk 1 above)
Comment (neihousaigaai) [reply json/player_score_list.json:1]: Oops, committed by accident.

**Review by nitro2 — APPROVED** [review]

**Pull request merged**